pkg/config/kube/conversion.go

```diff
@@ -60,6 +60,9 @@ func ConvertProtocol(port int32, portName string, proto corev1.Protocol, appProt
 		// "http2 over cleartext", which is also what our HTTP2 port is
 		case "kubernetes.io/h2c":
 			return protocol.HTTP2
+		// hbone is a protocol used internally
+		case "hbone":
```

Comment:
I'm not entirely sure if this will have effects on other parts, as it's used by `ConvertService`. Could @istio/wg-networking-maintainers confirm this? I think this will not affect waypoint service? Also, I didn't revise 
https://github.com/istio/istio/blob/6553f25d3b305a6ec01011c479acc904757f4133/pkg/config/protocol/instance.go#L65C1-L65C1
so validate will still report errors if users specify hbone as protocol.

Reply:
please donot invent a new appProtocol. `kubernetes.io/h2c` was for k8s convention

Reply:
Thanks, I think I didn't fix the issue in the right layer. The service can actually be filtered since it's not managed by users.

Reply:
HBONE is not h2c fwiw, it is HTTP2 over TLS